corehq/apps/locations/views.py

```diff
@@ -222,6 +227,23 @@ def to_json(loc):
             return [to_json(user.get_sql_location(self.domain))]
 
 
+@location_safe
+class FilteredLocationDownload(BaseLocationView):
+    urlname = 'filter_and_download_locations'
+    page_title = ugettext_noop('Filter and Download Locations')
+    template_name = 'locations/filter_and_download.html'
+
+    @method_decorator(require_can_edit_or_view_locations)
```

Comment:
nit: you can stick this decorator on the class with the arg `name='dispatch'` and not need to override dispatch here.